src/components/atoms/hero/components/Hero.spec.js

```diff
@@ -24,20 +25,11 @@ describe('Hero', () => {
     expect(queryByTestId('child-component')).toBeTruthy();
   });
 
-  describe('snapshot tests', () => {
-    it('renders with default props', () => {
-      const { container } = render(<Hero src="some-src" />);
-      expect(container.firstChild).toMatchSnapshot();
-    });
-
-    it('renders with custom props', () => {
-      const ChildComponent = () => <div data-testid="child-component" />;
-      const { container } = render(
-        <Hero src="some-src" height="240px" data-testid="some-testid">
-          <ChildComponent />
-        </Hero>
-      );
-      expect(container.firstChild).toMatchSnapshot();
-    });
+  it('renders the Hero component with a custom data-testid', () => {
```

Comment:
what's the point of this test, shouldn't it be in the first test. A test id doesn't have any functional value outside of here

Reply:
So I think I agree in part. The point of the test is to ensure that custom props will be passed down to the rendered element correctly. However, I believe that using something more obvious than a `testid`, such as a `data-some-custom-attribute` would demonstrate this better. I'll update this